mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.cpp

```diff
@@ -264,38 +265,17 @@ int CLeaderboardsTimes::TryAddAvatar(const uint64 &steamid, CUtlMap<uint64, int>
 void CLeaderboardsTimes::FillLeaderboards(bool bFullUpdate)
 {
     m_iSectionId = 0;
-
-    // Times
-    KeyValuesAD kvPlayerData("playdata");
-    GetPlayerTimes(kvPlayerData, bFullUpdate);
+    // Update times if needed
+    FillPlayerTimes(bFullUpdate);
 
     if (m_pCurrentLeaderboards && m_pTop10Leaderboards && m_pAroundLeaderboards
         && m_pLocalLeaderboards && m_pFriendsLeaderboards)
     {
-        SetVisible(false);
```

Comment:
You can remove the `SetVisible(true)` that went with this at the bottom of this func.